src/login/login_auth.cpp

```diff
@@ -273,6 +272,87 @@ int32 login_parse(int32 fd)
                 do_close_login(sd, fd);
             }
             break;
+        case LOGIN_CHANGE_PASSWORD:
+        {
+            const char* fmtQuery = "SELECT accounts.id,accounts.status \
+                                    FROM accounts \
+                                    WHERE accounts.login = '%s' AND accounts.password = PASSWORD('%s')";
+            int32 ret = Sql_Query(SqlHandle, fmtQuery, escaped_name, escaped_pass);
+            if (ret == SQL_ERROR || Sql_NumRows(SqlHandle) == 0)
+            {
+                session[fd]->wdata.resize(1);
+                ref<uint8>(session[fd]->wdata.data(), 0) = LOGIN_ERROR;
+                ShowWarning("login_parse: unexisting user" CL_WHITE"<%s>" CL_RESET" tried to connect\n", escaped_name);
```

Comment:
Technically this warning can be triggered on either the account not existing _or_ a password mismatch

Reply:
Fair point. Let me adjust the wording of this error to be more clear.

Reply:
For context, this was copied from this line: 
https://github.com/project-topaz/topaz/blob/821bcfe6b8db045a03694d032bde530e8152585f/src/login/login_auth.cpp#L205

It suffers from the same ambiguity problem.

Reply:
A lot of code in this file can be cleaned up and optimized. I tried to make as minimal changes as possible to reduce the risk of introducing a regression bug.

When I get some time I can try to clean up this file and provide a bit more structure and better organization for maintainability.

Reply:
Updated the wording here. Please take another look and let me know what you think.

Reply:
New wording looks good~! 👍